src/force/LennardJonesMolecule.cpp

```diff
@@ -0,0 +1,32 @@
+#include "LennardJonesMolecule.h"
+
+#include <cmath>
+#include <spdlog/spdlog.h>
+#include "utils/ArrayUtils.h"
+
+LennardJonesMolecule::LennardJonesMolecule(double epsilon_, double sigma_)
+    : epsilon(epsilon_), sigma(sigma_) {
+    epsilon_24 = epsilon * (-24);
+}
+
+std::array<double, 3> LennardJonesMolecule::calculateForce(Particle& p1,
+                                                           Particle& p2) const {
+    double distance = ArrayUtils::L2Norm(p1.getX() - p2.getX());
+
+    double summand_6 = std::pow((sigma / distance), 6);
+    double summand_12 = std::pow(summand_6, 2);
+
+    std::array<double, 3> force = (epsilon_24 / std::pow(distance, 2)) *
+                                  (summand_6 - (2 * summand_12)) *
+                                  (p1.getX() - p2.getX());
+
+#ifndef NO_LOG
```

Comment:
I wanted to extend it to everything else that logs to make you able to compile without logging at all but only added it here because this is relevant for the actual calculation. Going to extend it to all non-critical logs (probably).

Reply:
And have the neat features of the runtime logging.